Index: lib/Model/GetCharactersCharacterIdNotifications200Ok.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- lib/Model/GetCharactersCharacterIdNotifications200Ok.php	(date 1571590312000)
+++ lib/Model/GetCharactersCharacterIdNotifications200Ok.php	(date 1571590379061)
@@ -387,6 +387,7 @@
     const TYPE_TRANSACTION_REVERSAL_MSG = 'TransactionReversalMsg';
     const TYPE_TUTORIAL_MSG = 'TutorialMsg';
     const TYPE_WAR_ADOPTED = 'WarAdopted';
+    const TYPE_WAR_ADOPTED2 = 'WarAdopted ';
     const TYPE_WAR_ALLY_INHERITED = 'WarAllyInherited';
     const TYPE_WAR_ALLY_OFFER_DECLINED_MSG = 'WarAllyOfferDeclinedMsg';
     const TYPE_WAR_CONCORD_INVALIDATES = 'WarConcordInvalidates';
@@ -618,6 +619,7 @@
             self::TYPE_TRANSACTION_REVERSAL_MSG,
             self::TYPE_TUTORIAL_MSG,
             self::TYPE_WAR_ADOPTED,
+            self::TYPE_WAR_ADOPTED2,
             self::TYPE_WAR_ALLY_INHERITED,
             self::TYPE_WAR_ALLY_OFFER_DECLINED_MSG,
             self::TYPE_WAR_CONCORD_INVALIDATES,
